README.md

```diff
@@ -2,6 +2,11 @@
 
 > This Ruby on Rails capstone project is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
+## Video Description By Developer
+[Demo Video](https://www.loom.com/share/0812b02d73744cb29b3b1f4e754a7614)
+
+## Deployment
+[![Deployed on heroku](https://img.shields.io/badge/Deployed-Heroku-blueviolet)](https://fathomless-tundra-27272.herokuapp.com/)
```

Comment:
- [ ] Instead of hiding the live demo link in a badge, kindly add it as a plain link in the documentation for easier accessibility.

- [ ] Your live demo and your local app are quite different. Perhaps there is a mix-up somewhere. Kindly ensure that your live demo matches the local app.

  Live demo homepage | Local app homepage
  :---:|:---:
  ![jeff-live](https://user-images.githubusercontent.com/36542986/171865215-58e98ee8-fceb-48be-a22e-23f365484bc5.jpg) | ![jeff-local](https://user-images.githubusercontent.com/36542986/171865220-f03b974b-ef55-4c2e-9f32-4504221c991a.jpg)